README.md

```diff
@@ -64,6 +64,15 @@ new ExtractTextPlugin(options: filename | object)
 * `options.allChunks: boolean` extract from all additional chunks too (by default it extracts only from the initial chunk(s))
 * `options.disable: boolean` disables the plugin
 * `options.id: string` Unique ident for this plugin instance. (For advanced usage only, by default automatically generated)
+* `filenamefilter` function to modify path and filename before files are emitted
+``` javascript
+{
+  filenamefilter: function(filename) {
```

Comment:
`filenamefilter` => `filter` or why not just letting `options.filename` accepts a function ?